testing/python/fixtures.py

```diff
@@ -102,60 +104,62 @@ def test_funcarg_lookupfails(self, testdir):
             """
         )
 
-    def test_detect_recursive_dependency_error(self, testdir):
-        testdir.copy_example()
-        result = testdir.runpytest()
+    def test_detect_recursive_dependency_error(self, pytester: Pytester) -> None:
+        pytester.copy_example()
+        result = pytester.runpytest()
         result.stdout.fnmatch_lines(
             ["*recursive dependency involving fixture 'fix1' detected*"]
         )
 
-    def test_funcarg_basic(self, testdir):
-        testdir.copy_example()
-        item = testdir.getitem(Path("test_funcarg_basic.py"))
-        item._request._fillfixtures()
-        del item.funcargs["request"]
-        assert len(get_public_names(item.funcargs)) == 2
-        assert item.funcargs["some"] == "test_func"
-        assert item.funcargs["other"] == 42
+    def test_funcarg_basic(self, pytester: Pytester) -> None:
+        pytester.copy_example()
+        item = pytester.getitem(Path("test_funcarg_basic.py"))
+        item._request._fillfixtures()  # type: ignore[attr-defined]
```

Comment:
Can we add an `assert istinstance(item, <whatever it is>)` here to get rid of the type-ignores?

Reply:
added assertions